galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/verycommon/QueryParameters.java

```diff
@@ -171,6 +172,33 @@ public List<String> getResultsFromParameters (@NotNull List<String> rasResults)
 			}
 			return returnResults;
 		}
+		return null;
+	}
+// make func to validate status values
+	public List<String> getStatusesFromParameters () throws InternalServletException{
```

Comment:
This doesn't belong here. If you want to subclass QueryParameters to provide RAS-service-specific methods, then do that, but the idea is that things in the verycommon package will eventually be used by non-RAS servlets.